community/ibm-node-red-dev/templates/deployment.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "nodered.fullname" . }}
+  labels:
+    app: {{ template "nodered.name" . }}
+    chart: {{ template "nodered.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "nodered.name" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "nodered.name" . }}
+        release: {{ .Release.Name }}
+        chart: {{ template "nodered.chart" . }}
+        heritage: {{ .Release.Service }}
+      annotations:
+        productName: node_red
+        productID: node_red_{{ .Chart.Version }}
+        productVersion: {{ .Chart.Version }}
+      {{- if .Values.annotations }}
```

Comment:
The `annotations` value was removed from values.yaml.  If this is still needed, add back to `values.yaml` and `values-metadata.yaml`